trame_vtk/modules/vtk/addon_serializer.py

```diff
@@ -338,6 +338,33 @@ def cubeAxesSerializer(parent, actor, actorId, context, depth):
     }
 
 
+def imagedataSerializer(
+    parent, dataset, datasetId, context, depth, requested_fields=["Normals", "TCoords"]
+):
+    if hasattr(dataset, "GetDirectionMatrix"):
+        direction = [dataset.GetDirectionMatrix().GetElement(0, i) for i in range(9)]
+    else:
+        direction = [1, 0, 0, 0, 1, 0, 0, 0, 1]
+
+    # Extract dataset fields
+    fields = []
+    extractRequiredFields(fields, parent, dataset, context, "*")
+
+    return {
+        "parent": render_window_serializer.getReferenceId(parent),
+        "id": datasetId,
+        "type": render_window_serializer.class_name(dataset),
+        "properties": {
+            "spacing": dataset.GetSpacing(),
+            "origin": dataset.GetOrigin(),
+            "dimensions": dataset.GetDimensions(),
```

Comment:
@jourdain, do I remove this? It's working as expected with these changes, I think...

Reply:
yes remove dimensions.